src/runners/express.js

```diff
@@ -64,7 +102,12 @@ function finishRunner(expressEvent, res, req, startTime) {
     }
 
     if (req.route) {
-        eventInterface.addToMetadata(expressEvent, { route_path: req.route.path });
+        if (req.route.path instanceof Array) {
+            eventInterface.addToMetadata(expressEvent,
+                { route_path: findCalledParameteredPath(req) });
+        } else {
+            eventInterface.addToMetadata(expressEvent, { route_path: req.route.path });
```

Comment:
does this solve the case of nested routes?

Reply:
Do we want to show the full path in this case?